subjects/DPxAI/i-win-arguments/README.md

```diff
@@ -0,0 +1,158 @@
+## I Win Arguments
+
+> Mindful AI mode
+
+### Context
+
+You made it to the last mission in getting your full power, to make your robot alive and fully functional!
+
+The final step involves mastering the use of `arguments` in functions. By learning how to use and manage these `arguments` effectively, you can unlock the full potential of your robot and make it later truly come alive.
+
+Let's find out more!
+
+### AI-Powered Learning Techniques
+
+`Code Chunking Technique:`
+
+This type of prompt encourages you to break down larger pieces of code into smaller, digestible chunks.
+
+Each chunk is explained individually, allowing you to understand the purpose and functionality of each part.
+
+Find the examples across the subject ;)
+
+### Concepts
+
+### One Argument
+
+We mentioned it before with methods, functions can take arguments. They are always in between parens `()`.
+
+Let's use the same examples that we used for function calls:
+
+Remember this example of function call ?
+
+```js
+//       ↙ method
+console.log("Hello There !"); //<-
+//                 ↖ The String 'Hello There!' is
+//                   the argument of console.log()
+```
+
+We are now going to adapt `myFirstFuntion` so that it takes one argument : `arg1`.
+
+```js
+let myFirstFunction = (arg1) => {
+  //<-arg1 is inputed in between the parens
```

Comment:
same here `parens` should be `parenthesis`

Reply:
Addressed.